### PR TITLE
LPS-139217 Test Autom assert in trial button

### PR DESCRIPTION
**Pull request by lucassantanasantos**
Jira Tickets:
https://issues.liferay.com/browse/LPS-139217
LPS-105216

Motivation: 
Display an overview of the content in the performance panel

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 1]
```diff
@@ -89,21 +89,90 @@ definition {
...
+		task ("Open Content Performance") {
+
+			var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+			Navigator.openURL();
+
+			AnalyticsCloud.teardownAnalyticsCloud();
+
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:112]: Why are you deleting the Test Site Name site? Why is it needed?

File: portal-web/test/functional/com/liferay/portalweb/macros/ContentPerformance.macro [hunk 2]
```diff
@@ -3,5 +3,18 @@ definition {
 	macro openContentPerformance {
 		Click(locator1 = "ContentPerformance#CONTENT_PERFORMANCE_BUTTON");
 	}
+	
+	macro contentPerformanceMessage {
+		AssertTextEquals(
+			key_message = "${message}",
+			locator1 = "ContentPerformance#CONTENT_PERFORMANCE_MESSAGE",
+			value1 = "${message}");
+	}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/macros/ContentPerformance.macro]: There's no need to create a macro to assert a message, you can call the Assert method in the .testcase file.

File: portal-web/test/functional/com/liferay/portalweb/macros/ContentPerformance.macro [hunk 3]
```diff
@@ -3,5 +3,18 @@ definition {
...
 
+	macro contentPerformanceWithoutMessage {
+		AssertElementNotPresent(
+			key_message = "${message}",
+			locator1 = "ContentPerformance#CONTENT_PERFORMANCE_MESSAGE",
+			value1 = "${message}");
+	}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/macros/ContentPerformance.macro:15]: There's no need to create a macro to assert a message, you can call the Assert method in the .testcase file.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 4]
```diff
@@ -89,21 +89,90 @@ definition {
...
+		task ("Go to DXP portal, login and create site") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+
+			Navigator.openURL();
+
+			JSONGroup.addGroup(groupName = "Test Site Name");
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase]: There's no need to connect DXP with AC, the goal of this test is just to assert the messages when DXP is not connected to AC.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 5]
```diff
@@ -89,21 +89,90 @@ definition {
...
 
+		task ("Create content page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Content Page",
+				type = "content");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/content-page");
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase]: There's no need to create a Content Page.

**Comment by CarlosBrichete** [comment]
Also, it seems that the branch has some conflicts, please rebase with master

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase (hunk 5 above)
Comment (lucassantanasantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase]: ok Carlos, I fix this here.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase (hunk 4 above)
Comment (lucassantanasantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase]: I thought a lot about this test, how best to do it. but all other tests connect in ac, so it's interesting that the synchronization between ac and dxp happens in set up. this test is an exception and it was necessary to make this move

**Comment by CarlosBrichete** [comment]
ci:stop

**Comment by CarlosBrichete** [comment]
ci:test:tango-analytics-cloud

**Comment by mateusralv** [comment]
ci:test:tango-analytics-cloud

**Comment by CarlosBrichete** [comment]
ci:stop

**Comment by CarlosBrichete** [comment]
ci:stop

**Comment by CarlosBrichete** [comment]
ci:close

**Pull request closed without merging**